static/js/admin.js

```diff
@@ -118,6 +118,11 @@ function populate_streams(streams_data) {
     loading.destroy_indicator($('#admin_page_streams_loading_indicator'));
 }
 
+exports.toggle_email_change_display = function () {
+    $("#change_email").toggle();
+    $(".icon-vector-question-sign").toggle();
```

Comment:
The `".icon-vector-question-sign"` selector is way too broad -- this will toggle every `?` in the entire app!  

You should make sure you that the jQuery selector you're using here only covers the target element.  Please be careful about this in the future!  This kind of thing is a common source of hard-to-track-down bugs.